src/executor/attach_op_execs_pass.cc

```diff
@@ -0,0 +1,171 @@
+/*!
+ * Copyright (c) 2016 by Contributors
+ * \file attach_op_execs_pass.cc
+ * \brief Operator executor to execute each operator.
+ */
+#include <mxnet/base.h>
+#include <mxnet/operator.h>
+#include <mxnet/op_attr_types.h>
+#include <nnvm/graph_attr_types.h>
+#include "./exec_pass.h"
+
+namespace mxnet {
+
+namespace op {
+const OperatorProperty* OpPropGetOpProperty(const NodeAttrs& attrs);
+}  // namespace op
+
+namespace exec {
+
+// forward executor
+class ForwardOpExecutor : public OpExecutor {
```

Comment:
It is helpful to unifies the Operator and FCompute interface. We can discuss if we want to move it to registry level